### PR TITLE
ACCUMULO-4744 Fixed RFile API scanner bug

### PR DESCRIPTION
**Pull request by keith-turner**

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by milleruntime** [comment]
@keith-turner I can take a look at this.  I want to checkout your branch though so I can see the bug.

File: core/src/main/java/org/apache/accumulo/core/file/blockfile/impl/CachableBlockFile.java [hunk 1]
```diff
@@ -147,7 +147,7 @@ public long getStartPos() throws IOException {
   public static class Reader implements BlockFileReader {
     private final RateLimiter readLimiter;
     private BCFile.Reader _bc;
-    private String fileName = "not_available";
```
Comment (milleruntime) [inline core/src/main/java/org/apache/accumulo/core/file/blockfile/impl/CachableBlockFile.java]: OK I see why you made the changes but this old fileName does make me curious as to why "not_available" was chosen... any idea?

**Comment by keith-turner** [comment]
Merged in ed313f7

**Pull request closed without merging**